ert_gui/main_window.py

```diff
@@ -95,26 +95,16 @@ def __createMenu(self):
             help_link_item.triggered.connect(functools.partial(webbrowser.open, link))
 
 
-    def __quit(self):
-        self.__saveSettings()
-        self._clear_global_state()
-        qApp.quit()
-
-
     def __saveSettings(self):
         settings = QSettings("Equinor", "Ert-Gui")
         settings.setValue("geometry", self.saveGeometry())
         settings.setValue("windowState", self.saveState())
 
 
-    def _clear_global_state(self):
```

Comment:
The import of `ert_shared` is no longer necessary in this file and can be removed.